app/dts/bindings/zmk,ext-power-nicenano.yaml

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+#
+
+description: |
+  Allows controlling the external power output of nice!nano
+
+compatible: "zmk,ext-power-nicenano"
```

Comment:
We don't need a different *binding* for this at all. We'll just need it in the hardware description of the `nice_nano.dts` file where it would configure the generic driver.